.github/workflows/code-checks.yml

```diff
@@ -45,10 +47,10 @@ jobs:
 
     - name: Install dependencies
       run: |
-        pip install mypy==0.982  # TODO: pull this verion deom requirements/test.txt
-        pip install torch==1.12 --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
-        python ./requirements/pytorch/adjust-versions.py requirements/pytorch/extra.txt
-        # todo: adjust requirements for both code-bases
+        pip install mypy==0.982
```

Comment:
shall this be part of the requirements so it is managed by dependabot?

Reply:
I suggest not. It requires manual fixes for basically all releases

Reply:
yes, but this is still under PR so no merge until it is resolved

> for basically all releases

not sure what releases are you talking about... :otter:

Reply:
mypy releases

Reply:
we can skip, but at least we wont be frozen for ever :)